app/src/main/res/layout/no_internet_listitem.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout
+        xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:card_view="http://schemas.android.com/apk/res-auto"
+        xmlns:app="http://schemas.android.com/tools" app:ignore="NamespaceTypo">
+
+    <data>
+        <import type="android.view.View"/>
+        <variable name="article" type="com.androidtitan.culturedapp.model.newyorktimes.Article"/>
+        <variable name="multimedium" type="com.androidtitan.culturedapp.model.newyorktimes.Multimedium" />
+    </data>
```

Comment:
The user variable within data describes a property that may be used within this layout.